gpt4all-chat/metadata/models3.json

```diff
@@ -10,10 +10,9 @@
     "parameters": "7 billion",
     "quant": "q4_0",
     "type": "Mistral",
-    "description": "<strong>Best overall fast chat model</strong><br><ul><li>Fast responses</li><li>Chat based model</li><li>Trained by Mistral AI<li>Finetuned by NousResearch on the OpenHermes-2.5 dataset<li>Licensed for commercial use</ul>",
+    "description": "<strong>Best overall fast chat model</strong><br><ul><li>Fast responses</li><li>Chat based model</li><li>Accepts system prompts in ChatML format</li><li>Trained by Mistral AI<li>Finetuned by Nous Research on the OpenHermes-2.5 dataset<li>Licensed for commercial use</ul>",
     "url": "https://huggingface.co/NousResearch/Nous-Hermes-2-Mistral-7B-DPO-GGUF/resolve/main/Nous-Hermes-2-Mistral-7B-DPO.Q4_0.gguf",
     "promptTemplate": "<|im_start|>user\n%1<|im_end|>\n<|im_start|>assistant\n%2<|im_end|>\n",
```

Comment:
If you do not specify the systemPrompt key, this trailing comma will cause the JSON to fail to parse, but also GPT4All will use the default system prompt:
```
### System:
You are an AI assistant who gives a quality response to whatever humans ask of you.
```
which is probably not what you want (especially since it's not in ChatML syntax).

```suggestion
    "promptTemplate": "<|im_start|>user\n%1<|im_end|>\n<|im_start|>assistant\n%2<|im_end|>\n",
    "systemPrompt": ""
```